script/upgrades/MU05/MU05Checks.verify.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { console2 as console } from "forge-std/Script.sol";
+import { FixidityLib } from "mento-core-2.2.0/common/FixidityLib.sol";
+
+import { Reserve } from "mento-core-2.2.0/swap/Reserve.sol";
+import { TradingLimits } from "mento-core-2.2.0/libraries/TradingLimits.sol";
+
+import { MU05ChecksBase } from "./MU05Checks.base.sol";
+import { MU05Config, Config } from "./Config.sol";
+
+/**
+ * @title IBrokerWithCasts
+ * @notice Interface for Broker with tuple -> struct casting
+ * @dev This is used to access the internal trading limits
+ * config as a struct as opposed to a tuple.
+ */
+interface IBrokerWithCasts {
+  function tradingLimitsConfig(bytes32 id) external view returns (TradingLimits.Config memory);
+}
+
+contract MU05ChecksVerify is MU05ChecksBase {
+  using TradingLimits for TradingLimits.Config;
+  using FixidityLib for FixidityLib.Fraction;
+
+  constructor() public {
+    setUp();
+  }
+
+  function run() public {
+    console.log("\nStarting MU05 checks:");
+    MU05Config.MU05 memory config = MU05Config.get(contracts);
+
+    verifyReserveCollateralAssets();
+    verifyTradingLimits(config);
+  }
+
+  function verifyReserveCollateralAssets() internal {
+    console.log("\n== Verifying Reserve Collateral Assets ==");
+
+    require(Reserve(reserveProxy).isCollateralAsset(nativeUSDC), "❗️❌ Reserve collateral asset not set correctly");
+
+    // mint some native USDC to the main reserve in order to verify spending ratios
+    deal(nativeUSDC, reserveProxy, 100e6);
+    verifyCollateralSpendingRatio(nativeUSDC, 1e24);
+    console.log("🟢 Asset: %s successfully added to collateral asset list", nativeUSDC);
```

Comment:
move this below the require statement in L43?

Reply:
the idea was that it is successfully added if it is both a collateral asset and the spending ratio is set correctly.